README.md

```diff
@@ -356,6 +356,96 @@ countItems := response.CountItems
 count := response.Count
 ```
 
+## Importing and Exporting Data
+
+### Importing Data
+
+If you have large amounts of data, it is more efficient to import it to Pilosa instead of several `SetBit` queries. This library supports importing bits of a frame. The target frame must have been created before hand.
+
+Before starting the import, create an instance of a struct which implements `BitIterator` and pass it to `client.ImportFrame` function. This library ships with `CSVBitIterator` struct which supports importing bits in the CSV (comma separated values) format:
+```
+ROW_ID,COLUMN_ID
+```
+
+Optionally, a timestamp with GMT time zone can be added:
+```
+ROW_ID,COLUMN_ID,TIMESTAMP
+```
+
+Note that, each line corresponds to a single bit and the lines end with a new line (`\n` or `\r\n`).
```

Comment:
Note that each line corresponds to a single bit and ends with a new line (`\n` or `\r\n`).